.gitignore

```
@@ -35,6 +35,7 @@
 # VSCode
 #
 .vscode/*
+.vscode
```

```suggestion
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
!.vscode/*.code-snippets
```
Use the correct ignores